js/modal.js

```diff
@@ -259,11 +260,12 @@
 
   Modal.prototype.setScrollbar = function () {
     var bodyPad = parseInt((this.$body.css('padding-right') || 0), 10)
+    this.originalBodyPad = document.body.style.paddingRight || null
```

Comment:
What about `|| ''` instead of `|| null`, and then getting rid of the ternary below?